core/sequencer/server.go

```diff
@@ -103,11 +103,11 @@ func NewServer(
 		tlog:      tlog,
 		tmap:      tmap,
 		mutations: mutations,
-		queue:     queue,
+		logs:      logs,
 	}
 }
 
-// RunBatch reads mutations out of the queue and calls CreateEpoch.
+// RunBatch reads mutations out of the logs and calls CreateEpoch.
```

Comment:
unrelated, but I think it'd help the reader if this method comment talked a bit about what and why rather than (or perhaps in addition to if you feel it's important) how.